packages/executor/package.json

```diff
@@ -8,7 +8,7 @@
     "dist/*"
   ],
   "scripts": {
-    "start": "ts-node ./src/index.ts",
+    "start": "hardhat run ./src/index.ts",
```

Comment:
This is the same as running ts-node, except the HRE is also injected into the environment, allowing us to use it for verification